src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,143 @@
 package core.basesyntax;
 
+import static java.util.Objects.hash;
+
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int MAXIMUM_CAPACITY = 1073741824;
```

Comment:
What is the purpose of this parameter?

Reply:
The maximum size of an array equal to 2^30 if multiplied by GROW_CAPACITY_FACTOR the size of the value will go beyond the int max value